tests/spoke/service_import_test.go

```diff
@@ -151,6 +153,37 @@ var _ = Describe("ServiceImportController", func() {
 				}
 				return true
 			}, time.Second*30, time.Millisecond*250).Should(BeTrue())
+
```

Comment:
the diff is confusing. can you close and reopen the pr to make sure the diff is correct?